tests/unit/topology/CMakeLists.txt

```diff
@@ -14,7 +14,9 @@ if (HPX_WITH_SHARED_PRIORITY_SCHEDULER AND HPX_WITH_CXX14_RETURN_TYPE_DEDUCTION)
     THREADS_PER_LOCALITY -2
     ARGS --size=128 --nb=512 --tiles-per-domain=2 --col-proc=1 --row-proc=1)
 endif()
-set(numa_allocator_FLAGS COMPONENT_DEPENDENCIES iostreams)
+set(numa_allocator_FLAGS
+  COMPONENT_DEPENDENCIES iostreams
+  DEPENDENCIES hpx_topology)
```

Comment:
This is probably a leftover from my changes but I don't think we need to have `hpx_topology` as dependency here. We do `add_hpx_executable` without `NOLIBS` so we should get it automatically through `hpx` as a dependency.